db/schema.rb

```diff
@@ -71,20 +66,10 @@
     t.string   "checksum",         :limit=>32
     t.integer  "time_spent_total", :default=>0
     t.integer  "release_id",       :index=>{:name=>"index_grouped_issues_on_release_id"}, :foreign_key=>{:references=>"releases", :name=>"fk_grouped_issues_release_id", :on_update=>:restrict, :on_delete=>:cascade}
+    t.boolean  "muted",            :default=>false
   end
   add_index "grouped_issues", ["website_id", "checksum"], :name=>"index_grouped_issues_on_website_id_and_checksum", :unique=>true
 
-  create_table "integrations", force: :cascade do |t|
-    t.integer  "website_id",    :index=>{:name=>"index_integrations_on_website_id"}, :foreign_key=>{:references=>"websites", :name=>"fk_rails_ef5f282bb0", :on_update=>:no_action, :on_delete=>:no_action}
-    t.string   "provider",      :null=>false
-    t.text     "configuration"
-    t.string   "name",          :null=>false
-    t.boolean  "disabled",      :default=>false
-    t.text     "error"
-    t.datetime "created_at",    :null=>false
-    t.datetime "updated_at",    :null=>false
-  end
```

Comment:
After rake db:drop, rake db:create, rake db:schema:load.. I have added the new migration, then I ran rake db:migrate.. 
That's when schema changed